packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -606,32 +524,43 @@
 					"value": 0
 				},
 				{
-					"label": "three attempts to turn on the load",
+					"label": "Three attempts to turn on the load",
 					"value": 1
 				}
 			]
 		},
 		"38": {
-			"label": "Brightness level correction for flickering loads",
+			"label": "Brightness Level Correction for Flickering Loads",
 			"valueSize": 2,
+			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 255,
-			"defaultValue": 255
+			"defaultValue": 255,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Always enable",
+					"value": 255
+				}
+			]
 		},
 		"39": {
-			"label": "Power limit - OVERLOAD",
+			"label": "Power Limit - OVERLOAD",
```

Comment:
Nothing should be in all caps. For this specific one usually we just call the parameter Overcurrent Protection, or just Power Limit.

Reply:
Removed "overload" from here and parameter 45